deploy/service_account.yaml

```diff
@@ -0,0 +1,5 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: apiserver
```

Comment:
name `apiserver` looks generic.

Reply:
Done in https://github.com/redhat-developer/git-service/pull/1/commits/b11eebe97c974ff98c10e4b8a7f97c3f5d82021f